etc/docs-main.md

```diff
@@ -0,0 +1,7 @@
+# swift-bson Documentation
```

Comment:
I realized that, since we are generating the docs on master, this has to live on master too, oops